client/src/app/pages/assessment-management/questionnaire/components/answer-table.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Table, Tbody, Td, Th, Thead, Tr } from "@patternfly/react-table";
+
+import { useLocalTableControls } from "@app/hooks/table-controls";
+import {
+  ConditionalTableBody,
+  TableHeaderContentWithControls,
+  TableRowContentWithControls,
+} from "@app/components/TableControls";
+import { NoDataEmptyState } from "@app/components/NoDataEmptyState";
+import { Answer } from "@app/api/models";
+import { Label, Text } from "@patternfly/react-core";
+import spacing from "@patternfly/react-styles/css/utilities/Spacing/spacing";
+import { IconedStatus, IconedStatusPreset } from "@app/components/IconedStatus";
+import { TimesCircleIcon } from "@patternfly/react-icons";
+import { WarningTriangleIcon } from "@patternfly/react-icons";
+export interface IWaveStatusTableProps {
+  answers: Answer[];
+}
+
+const AnswerTable: React.FC<IWaveStatusTableProps> = ({ answers }) => {
+  const { t } = useTranslation();
+
+  const tableControls = useLocalTableControls({
+    idProperty: "choice",
+    items: answers,
+    columnNames: {
+      choice: "Answer choice",
+      weight: "Weight",
+    },
+    hasActionsColumn: false,
+    hasPagination: false,
+    variant: "compact",
+  });
+  const {
+    currentPageItems,
+    numRenderedColumns,
+    propHelpers: { tableProps, getThProps, getTdProps },
+  } = tableControls;
+
+  const getIconByRisk = (risk: string): React.ReactElement => {
+    switch (risk) {
+      case "green":
+        return <IconedStatus preset="Ok" />;
+      case "unknown":
```

Comment:
nit really but could probably just let this fall into default

Reply:
Done.